vaadin-select-flow-demo/src/main/java/com/vaadin/flow/component/select/SelectView.java

```diff
@@ -89,6 +92,54 @@ private void disabledAndReadonly() {
         addCard("Disabled and read-only", layout);
     }
 
+    private void helperText() {
+        // begin-source-example
+        // source-example-heading: Helper text and component
+        Select<String> degree = new Select<>("Baccalaureate", "Licence",
+              "Master", "Doctorate");
+        degree.setLabel("Academic degree");
+        degree.setHelperText(
+              "Please, select only the highest accomplished degree");
+
+        Select<String> pet = new Select<>("Cat", "Dog", "Rabbit", "Fish",
+              "Bird", "Other");
+        pet.setLabel("Pets");
+        pet.setHelperComponent(new Span("Your favorite pet"));
+
+        add(degree, pet);
+        // end-source-example
+
+        degree.getStyle().set("margin-right", "5px");
+        HorizontalLayout layout = new HorizontalLayout(degree, pet);
+        layout.getStyle().set("flex-wrap", "wrap");
+        addCard("Helper text and component", layout);
+    }
+
+    private void helperTextAbove () {
+        // begin-source-example
+        // source-example-heading: Helper text and component above the field
+        Select<String> degree = new Select<>("Baccalaureate", "Licence",
+              "Master", "Doctorate");
+        degree.setLabel("Academic degree");
+        degree.setHelperText(
+              "Please, select only the highest accomplished degree");
+        degree.getElement().getThemeList().set("helper-above-field", true);
+
+        Select<String> pet = new Select<>("Cat", "Dog", "Rabbit", "Fish",
+              "Bird", "Other");
+        pet.setLabel("Pets");
+        pet.setHelperComponent(new Span("Your favorite pet"));
+        pet.getElement().getThemeList().set("helper-above-field", true);
+
+        add(degree, pet);
+        // end-source-example
+
+        degree.getStyle().set("margin-right", "5px");
+        HorizontalLayout layout = new HorizontalLayout(degree, pet);
+        layout.getStyle().set("flex-wrap", "wrap");
+        addCard("Helper text and component above the field", layout);
```

Comment:
I guess this demo could go to the "Theme variants" tab.

Reply:
Done :)